site/docs/main/config/tls-delegation.md

```diff
@@ -3,6 +3,8 @@
 In order to support wildcard certificates, TLS certificates for a `*.somedomain.com`, which are stored in a namespace controlled by the cluster administrator, Contour supports a facility known as TLS Certificate Delegation.
 This facility allows the owner of a TLS certificate to delegate, for the purposes of referencing the TLS certificate, permission to Contour to read the Secret object from another namespace.
 Delegation works for both HTTPProxy and Ingress v1beta1 resources.
+Ingress v1 is not currently supported due to changes in the spec that make this impossible.
```

Comment:
nit: Maybe add something that `TLSCertificateDelegation` is currently not supported with IngressV1. My first glance it read like Ingressv1 isn't supported, but we are in the TLSCertDelegation section.

Reply:
good point, yeah ill change that